internal/graph/graph.go

```diff
@@ -420,6 +421,35 @@ func newTree(prof *profile.Profile, o *Options) (g *Graph) {
 	return selectNodesForGraph(nodes, o.DropNegative)
 }
 
+var javaRegExp = regexp.MustCompile(`^(?:[a-z]\w*\.)*([A-Z][\w\$]*\.(?:<init>|[a-z]\w*(?:\$\d+)?))(?:(?:\()|$)`)
+var goRegExp = regexp.MustCompile(`^(?:[\w\-\.]+\/)+(.+)`)
+var cppAnonymousPrefixRegExp = regexp.MustCompile(`^(?:\(anonymous namespace\)::)(.*)`)
```

Comment:
I wonder if this regular expression can be merged into cppRegExp, as an optional prefix. Then the loop below would simplify.

Reply:
I tried merging it as an optional prefix, but then cases like "(anonymous namespace)::foo" are missed. 

I have a single working regex for now, I think.